### PR TITLE
add replace button to card

### PR DESCRIPTION
**Pull request by oleksandr-didyk**

### CODE REVIEW

File: WanderLost/WanderLost/Client/Components/Card.razor [hunk 1]
```diff
@@ -9,16 +9,19 @@
     </div>
   </div>
   <div class="m-card__buttons">
-    <CardButton Type="CardButton.Variant.Like" OnClick="@(async () => await SubmitVote(Merchant.Id, VoteType.Upvote))" Active="@(CurrentVote(Merchant.Id) == VoteType.Upvote)" />
+    <CardButton ClassName="m-card__like" Type="CardButton.Variant.Like" OnClick="@(async () => await SubmitVote(Merchant.Id, VoteType.Upvote))" Active="@(CurrentVote(Merchant.Id) == VoteType.Upvote)" />
     <div class="m-card__score">@Merchant.Votes</div>
-    <CardButton Type="CardButton.Variant.Dislike" OnClick="@(async () => await SubmitVote(Merchant.Id, VoteType.Downvote))" Active="@(CurrentVote(Merchant.Id) == VoteType.Downvote)" />
+    <CardButton ClassName="m-card__dislike" Type="CardButton.Variant.Dislike" OnClick="@(async () => await SubmitVote(Merchant.Id, VoteType.Downvote))" Active="@(CurrentVote(Merchant.Id) == VoteType.Downvote)" />
+    <CardButton ClassName="m-card__report" Title="Replace" Type="CardButton.Variant.Report" OnClick="@OnClickReport" />
```
Comment (user1736) [inline WanderLost/WanderLost/Client/Components/Card.razor]: Based on user vote existence title should be either "Submit correction" or "Suggest new location"

File: WanderLost/WanderLost/Client/Components/Card.razor [hunk 2]
```diff
@@ -41,11 +44,17 @@
...
+
+  [Parameter]
+  public string Region { get; set; } = default!;
+
   public string Title
   {
     get
     {
-      return Merchant.Name + " (" + Merchant.Zone + ")";
+      return $"{Region} - ({Merchant.Zone})";
```
Comment (user1736) [inline WanderLost/WanderLost/Client/Components/Card.razor:58]: discussed, `-` will be removed in a separate PR

File: WanderLost/WanderLost/Client/Components/Card.razor [hunk 3]
```diff
@@ -84,4 +93,11 @@
       return OnLocationClick is not null ? "m-card__location--active" : "";
     }
   }
+
+    public void OnClickReport()
```
Comment (user1736) [inline WanderLost/WanderLost/Client/Components/Card.razor]: method alignment is off :)

**Review by user1736 — CHANGES_REQUESTED** [review]

File: WanderLost/WanderLost/Client/Components/Card.razor (hunk 3 above)
Comment (oleksandr-didyk) [reply WanderLost/WanderLost/Client/Components/Card.razor]: Thanks for noticing, fixed

File: WanderLost/WanderLost/Client/Components/Card.razor (hunk 1 above)
Comment (oleksandr-didyk) [reply WanderLost/WanderLost/Client/Components/Card.razor]: Yeah, good idea, done

File: WanderLost/WanderLost/Client/Components/CardButton.razor [hunk 4]
```diff
@@ -26,4 +20,24 @@
 
   [Parameter]
   public Action OnClick { get; set; }
+
+  [Parameter]
+  public string ClassName { get; set; } = "button";
```
Comment (user1736) [inline WanderLost/WanderLost/Client/Components/CardButton.razor:25]: shouldn't have default value here (discussed offline)
